diff -ruN grpc-1.26.0.orig/setup.py grpc-1.26.0/setup.py
--- grpc-1.26.0.orig/setup.py	2019-12-18 02:29:27.000000000 +0100
+++ grpc-1.26.0/setup.py	2020-12-09 08:41:38.202678255 +0100
@@ -179,7 +179,7 @@
       # available dynamically
       EXTRA_ENV_COMPILE_ARGS += ' /MT'
   elif "linux" in sys.platform:
-    EXTRA_ENV_COMPILE_ARGS += ' -std=gnu99 -fvisibility=hidden -fno-wrapv -fno-exceptions'
+    EXTRA_ENV_COMPILE_ARGS += ' -fvisibility=hidden -fno-wrapv -fno-exceptions'
   elif "darwin" in sys.platform:
     EXTRA_ENV_COMPILE_ARGS += ' -stdlib=libc++ -fvisibility=hidden -fno-wrapv -fno-exceptions'
 
